BCStudentSoftwareDevTeam/celts
# Rework sidebar 237

This PR fixes issue #237 

## Summary:
The sidebar structure had become outdated due to new changes to development. Now the sidebar will have an admin drop-down that will include Activity Logs (prev. Admin Logs), Course Proposals (prev. Course Proposal Management), and Settings (prev. Admin Management). This drop-down is not accessible as a Student Admin.  

## To Test:
Go to the CELTS web app as an admin and you will see the admin dropdown and the options held under it. Pay close attention to what happens after you click as the tab will remain actively highlighted but the drop-down will close and hide the options this can easily be reopened but I am not sure your feelings regarding this. Also, the clickable area is noticeably smaller than the regular nav links. Not sure if these differences are okay or if you would like to approach this differently.

## Previous UI:
![image](https://user-images.githubusercontent.com/67805799/164081723-5bbe1386-89ed-42b0-a163-1cc72b18e2e7.png)
